integration/util.go

```diff
@@ -26,6 +28,14 @@ func getCortexProjectDir() string {
 		return dir
 	}
 
+	// use the git path if available
+	if gitPath, err := exec.LookPath("git"); err == nil {
+		dir, err := exec.Command(gitPath, "rev-parse", "--show-toplevel").Output()
```

Comment:
Nit: no need to do `LookPath` first. exec.Command(“git”, ...) will do that already, and possibly return `exec.ErrNotFound`.

Reply:
Good point, just pushed that change :+1: